### PR TITLE
added support for random food found inside game-world crates.

### PR DESCRIPTION
**Pull request by WillUGHandled**
Crates found in the game world previously didn't do anything, now they have random food items in them.

### CODE REVIEW

File: src/plugins/objects/crates/crates.plugin.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { objectInteractionActionHandler } from '@engine/world/action/object-interaction.action';
+import { World } from '@engine/world';
+import { itemIds } from '@engine/world/config/item-ids';
+import { LandscapeObject } from '@runejs/filestore';
+
+export const action: objectInteractionActionHandler = (details) => {
+    details.player.busy = true;
+    details.player.playAnimation(827);
+    var random = Math.floor(Math.random() * 3);  
+    var veggies = [itemIds.onion, itemIds.grain, itemIds.cabbage];
```
Comment (SchauweM) [inline src/plugins/objects/crates/crates.plugin.ts]: use const instead of var

File: src/plugins/objects/crates/crates.plugin.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    var pickedItem = findItem(veggies[random]);
```
Comment (SchauweM) [inline src/plugins/objects/crates/crates.plugin.ts]: use const instead of var

File: src/plugins/objects/crates/crates.plugin.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { findItem, widgets } from '@engine/config';
+import { objectInteractionActionHandler } from '@engine/world/action/object-interaction.action';
+import { World } from '@engine/world';
+import { itemIds } from '@engine/world/config/item-ids';
+import { LandscapeObject } from '@runejs/filestore';
+
+export const action: objectInteractionActionHandler = (details) => {
+    details.player.busy = true;
+    details.player.playAnimation(827);
+    var random = Math.floor(Math.random() * 3);
```
Comment (SchauweM) [inline src/plugins/objects/crates/crates.plugin.ts]: use const/let instead of var

**Review by SchauweM — CHANGES_REQUESTED** [review]

File: src/plugins/objects/crates/crates.plugin.ts (hunk 1 above)
Comment (WillUGHandled) [reply src/plugins/objects/crates/crates.plugin.ts]: updated in latest commit

File: src/plugins/objects/crates/crates.plugin.ts (hunk 2 above)
Comment (WillUGHandled) [reply src/plugins/objects/crates/crates.plugin.ts]: updated in last commit

**Review by Tynarus — DISMISSED** [review]
😍 :shipit:

**Review by SchauweM — APPROVED** [review]

**Pull request merged**